pages/library/_viewId.vue

```diff
@@ -1,10 +1,13 @@
 <template>
   <v-container>
-    <v-row>
+    <v-row v-if="items.length">
       <v-col cols="12" class="card-grid-container">
         <card v-for="item in items" :key="item.Id" :item="item" />
       </v-col>
     </v-row>
+    <v-row v-else justify="center">
+      <h1 class="text-h4 text-center">{{ $t('noItemsFound') }}</h1>
+    </v-row>
```

Comment:
We should probably distinguish between "There is nothing because this doesn't exist yet" and "This exists but is empty. 

Something like:
```suggestion
    <v-row :justify="items ? 'left': undefined">
      <v-skeleton-loader v-if="!items" /> <!-- Some kind of structure here to mimic the cards and give visual feedback -->
      <h1 v-else-if="!nodes.length" class="text-h4 text-center">{{ $t('noItemsFound') }}</h1>
      <v-col v-else cols="12" class="card-grid-container">
        <card v-for="item in items" :key="item.Id" :item="item" />
      </v-col>
    </v-row>
```

It's just pseudo-code to give an idea, but I think it makes it more clear to the user what is going on exactly.

Reply:
I've added another component called skeleton cards. Which uses several vuetify skeleton loaders to give the correct effect.